app/wama.web/Controllers/UserToolController.cs

```diff
@@ -239,7 +239,13 @@ public IActionResult AddCertification(string memberId)
             ViewBag.MemberId = memberId;
             return View($"{Constants.ADMIN_CONSOLE_USER_TOOL_DIRECTORY}/ViewCertification.cshtml", Certifications);
         }
-
+        public async Task<IActionResult> ViewOneCertification(string memberId) //need to straight idea first
```

Comment:
I was thinking of deleting that function but left it just in case, before i make any change on those, i think i need to talk about that function.
and ill change parameter